### ISSUE TITLE
Broken void

### ISSUE THREAD

**Issue by Junkberry** [FTBTeam/FTB-Sky-Odyssey#248]
* **Modpack Version**: 1.0.0

* **Issue**: Logged into the world seeing myself falling into the void. Can not fly up (angel ring) and neither do I die (I waited to -30 000). Also commands do not work 

* **Is it Repeatable?**: No (it might be but not sure how)

**Comment by Junkberry** [comment]
After some tries, I found some things that might be helpful (idk).

- I tried a simple command such as help, but that didn't work.
- The pack has FTB Utils but /spawn doesnt work.
- I also tried changing my PlayerPosition using NBT to 0, 0 at 100 height but instead I spawn at -14 -13 at 0 height and start to fall down again.

I really don't want to start over, It would be great if someone could help :)
